docker/scout/scripts/addresses.py

```diff
@@ -491,6 +487,9 @@
         "B_80_BAL_20_WETH": "0x5c6ee304399dbdb9c8ef030ab642b10820db8f56",
         "Bstable_auraBAL": "0x3dd0843a028c86e0b760b1a76929d1c5ef93a2dd",
     },
+    "balancer_misc": {
```

Comment:
I think there is already a balancer section that has the vault and/or other misc stuff.  no?